src/elements/ix-video.ts

```diff
@@ -166,17 +166,15 @@ export class IxVideo extends LitElement {
     listener: EventListenerOrEventListenerObject,
     options?: boolean | EventListenerOptions
   ) => {
-    const event = DefaultVideoEventsMap[type];
```

Comment:
The reason `DefaultVideoEventsMap` is a map and not just an array of strings is to 1) avoid misspellings and 2) allow us to cheapy check for the presence of event names in the map in the future to validate listeners added by users.